src/arm/bbai-bone-buses.dtsi

```diff
@@ -468,8 +468,15 @@ bone_timer_5: &timer15 {
 
 // McASP
 bone_mcasp_0: &mcasp1 {
+        #sound-dai-cells= <0>;
 	assigned-clocks = <&ipu_clkctrl DRA7_MCASP1_CLKCTRL 24>;
 	assigned-clock-parents = <&sys_clkin2>; /* 22579200 Hz (see dra7xx-clocks.dtsi) */
+        serial-dir = <  /* 0: INACTIVE, 1: TX, 2: RX */
```

Comment:
These files are likely to be referenced a lot. Is there a reference to what each of these numbers refer? I knew it at one point, but I forget so easily. Just don't want blind copy-and-paste.

Reply:
I have no idea about these indexes, Niklas fixed these as you can see this contribution comes from him.

Reply:
Please discover and document.

https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/sound/davinci-mcasp-audio.txt

That provides a start, but it probably needs to be patched with something that indicates the I/O pin indexes. The point of the cape compatibility layer is to make it clear to people how they can create hardware using the interfaces that "just works" with the software. At least knowing what might or might not ever need to change would help.